gsa/src/web/pages/tasks/row.js

```diff
@@ -103,19 +103,25 @@ const Row = ({
 }) => {
   const {scanner, observers} = entity;
 
-  const obs = [];
+  const obs = {};
+
   if (isDefined(observers)) {
-    if (isString(observers)) {
-      obs.push(_('User {{name}}', {name: observers}));
-    } else {
-      if (isDefined(observers.role)) {
-        obs.push(_('Role {{name}}', {name: observers.role.name}));
-      }
-      if (isDefined(observers.group)) {
-        obs.push(_('Group {{name}}', {name: observers.group.name}));
-      }
+    if (isDefined(observers.user)) {
+      obs.user = _('Users ');
+      observers.user.forEach(user => (obs.user += user + ' '));
+      obs.user += '\n';
+    }
+    if (isDefined(observers.role)) {
+      obs.role = _('Roles ');
+      observers.role.forEach(role => (obs.role += role.name + ' '));
+      obs.role += '\n';
+    }
+    if (isDefined(observers.group)) {
+      obs.group = _('Groups ');
```

Comment:
```suggestion
      obs.group = _('Groups {{groups}}', {groups: observers.groups.join(' ')});
```

Reply:
I guess we should also display users, roles and groups with comma separation because names may contain spaces already.
Should be e.g. 
```javascript
observers.groups.join(', ')
```